From 12a317af8f5eabb12c4d189c8bda890f1d9a522c Mon Sep 17 00:00:00 2001
From: Bhawanpreet Lakha <Bhawanpreet.Lakha@amd.com>
Date: Fri, 28 Jul 2017 12:07:38 -0400
Subject: [PATCH 0705/4131] drm/amd/display: Move and Rename
 "is_stream_changed()"

 -Move "is_stream_changed()" to DC interface
 -Rename "is_stream_changed()" to "dc_is_stream_changed()"

Signed-off-by: Bhawanpreet Lakha <Bhawanpreet.Lakha@amd.com>
Reviewed-by: Harry Wentland <Harry.Wentland@amd.com>
Signed-off-by: Alex Deucher <alexander.deucher@amd.com>
---
 drivers/gpu/drm/amd/display/dc/core/dc.c          | 2 +-
 drivers/gpu/drm/amd/display/dc/core/dc_resource.c | 4 ++--
 drivers/gpu/drm/amd/display/dc/dc.h               | 2 ++
 drivers/gpu/drm/amd/display/dc/inc/resource.h     | 2 --
 4 files changed, 5 insertions(+), 5 deletions(-)

diff --git a/drivers/gpu/drm/amd/display/dc/core/dc.c b/drivers/gpu/drm/amd/display/dc/core/dc.c
index ba80933..5acf4c6 100644
--- a/drivers/gpu/drm/amd/display/dc/core/dc.c
+++ b/drivers/gpu/drm/amd/display/dc/core/dc.c
@@ -641,7 +641,7 @@ static bool is_validation_required(
 
 		if (set[i].surface_count != context->stream_status[i].surface_count)
 			return true;
-		if (!is_stream_unchanged(set[i].stream, context->streams[i]))
+		if (!dc_is_stream_unchanged(set[i].stream, context->streams[i]))
 			return true;
 
 		for (j = 0; j < set[i].surface_count; j++) {
diff --git a/drivers/gpu/drm/amd/display/dc/core/dc_resource.c b/drivers/gpu/drm/amd/display/dc/core/dc_resource.c
index 84efe72..4a1eaab 100644
--- a/drivers/gpu/drm/amd/display/dc/core/dc_resource.c
+++ b/drivers/gpu/drm/amd/display/dc/core/dc_resource.c
@@ -1155,7 +1155,7 @@ static bool are_stream_backends_same(
 	return true;
 }
 
-bool is_stream_unchanged(
+bool dc_is_stream_unchanged(
 	struct dc_stream *old_stream, struct dc_stream *stream)
 {
 
@@ -1176,7 +1176,7 @@ bool resource_validate_attach_surfaces(
 
 	for (i = 0; i < set_count; i++) {
 		for (j = 0; old_context && j < old_context->stream_count; j++)
-			if (is_stream_unchanged(
+			if (dc_is_stream_unchanged(
 					old_context->streams[j],
 					context->streams[i])) {
 				if (!resource_attach_surfaces_to_context(
diff --git a/drivers/gpu/drm/amd/display/dc/dc.h b/drivers/gpu/drm/amd/display/dc/dc.h
index 63fd5b3..0918569 100644
--- a/drivers/gpu/drm/amd/display/dc/dc.h
+++ b/drivers/gpu/drm/amd/display/dc/dc.h
@@ -534,6 +534,8 @@ struct dc_stream_update {
 	struct dc_transfer_func *out_transfer_func;
 };
 
+bool dc_is_stream_unchanged(
+	struct dc_stream *old_stream, struct dc_stream *stream);
 
 /*
  * Setup stream attributes if no stream updates are provided
diff --git a/drivers/gpu/drm/amd/display/dc/inc/resource.h b/drivers/gpu/drm/amd/display/dc/inc/resource.h
index 571bfae..ca3f493 100644
--- a/drivers/gpu/drm/amd/display/dc/inc/resource.h
+++ b/drivers/gpu/drm/amd/display/dc/inc/resource.h
@@ -132,8 +132,6 @@ struct pipe_ctx *find_idle_secondary_pipe(
 bool resource_is_stream_unchanged(
 	struct validate_context *old_context, struct dc_stream *stream);
 
-bool is_stream_unchanged(
-	struct dc_stream *old_stream, struct dc_stream *stream);
 
 bool resource_validate_attach_surfaces(
 		const struct dc_validation_set set[],
-- 
2.7.4

